NumberGuess.py

```diff
@@ -0,0 +1,23 @@
+#"Guess my Number" game
+#Takes user input of a number from 1-100 and compares it to a randomly-generated value
+#Tells user if their guess is higher or lower than the correct answer
+#User may keep guessing until they get the correct value
+
+import numpy
+range=numpy.arange(1,101) #define array of possible number choices to exclude 0 and include 100
+answer=numpy.random.choice(range) #generate a random number guess
+guess=0 #clear variable from the last game
+
+print("I'm thinking of a number from 1-100. Take a guess!") #Initiate game
+guess=int(input(prompt="Your guess:")) #Store a guess from the user
+
+while guess != answer: #Test if the user input is right
+    if guess < answer: #If not, test whether the guess is lower or higher than the answer
```

Comment:
The code looks good. You may want to put "guess=int(input(prompt="Your guess:"))" within the while loop so that you don't need to run the code after the first input